### ISSUE TITLE
Training with default "configs/faster_rcnn_r50_fpn_1x.py" only gets AP=35.6

### ISSUE THREAD

**Issue by bowenc0221** [open-mmlab/mmdetection#25]
Hi,
Thanks for sharing this great work.  

I tried your code to train a Faster RCNN FPN with ResNet-50 detector with this config file `configs/faster_rcnn_r50_fpn_1x.py` by running  
`python tools/train.py ./configs/faster_rcnn_r50_fpn_1x.py --gpus 4 --work_dir ./output --validate`  

And I tested the model with  
`python tools/test.py ./configs/faster_rcnn_r50_fpn_1x.py ./output/latest.pth --gpus 4 --out ./output/results.pkl --eval bbox`.  

I got this result:
```
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.356
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.571
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.382
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.204
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.393
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.450
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.304
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.489
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.515
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.331
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.556
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.644
```

I noticed the result (35.6) is lower than the one reported in the [MODEL_ZOO](https://github.com/open-mmlab/mmdetection/blob/master/MODEL_ZOO.md) (36.4).  

I also tested on the 11th epoch and the result is 35.0.  

Do you think it is normal? I don't think the variance should be this large.

**Comment by bowenc0221** [comment]
Thanks! @hellock 

I haven't read the code in detail, but I found the learning rate is not scaled if I simply change the number of GPUs. 

How is the loss normalized? I'm running another experiment with lr=0.01 (instead of 0.02 for 8 GPUs).

**Comment by hellock** [comment]
@bowenc0221 We tried the 4 GPU setting and got mAP=36.5. We use the default hyper parameters except that lr=0.01.

**Comment by bowenc0221** [comment]
Thanks! @hellock 
I also got 36.6 using 4 GPUs.
